eng/pipelines/scripts/generate_overview_from_readme.py

```diff
@@ -1,10 +1,10 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-# Use case: Given a README.md file, generate a readme_overview.html file and place it next 
+# Use case: Given a README.md file, generate a readme_overview.html file and place it next
```

Comment:
Please do not change this file or check this change in. There's already a PR that was investigating this and it doesn't work and breaks other things.

Reply:
Ok, I will restore the file.

Reply:
nit: if the only thing in this file are removal of two trailing spaces then please remove this entirely.

Reply:
Sorry, I forgot when and how I removed these two trailing spaces thus neglacting the changes. I will revert them.